src/main/java/lotto/domain/Ranks.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import static java.util.stream.Collectors.*;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+public class Ranks {
+    private final List<Rank> ranks;
+
+    public Ranks(List<Rank> ranks) {
+        this.ranks = ranks;
+    }
+
+    public Statistics makeStatistics() {
+        Map<Rank, Long> rankCounts = ranks.stream()
+            .collect(groupingBy(Function.identity(), counting()));
+
+        EarningRate earningRate = Money.calculateEarningRate(
```

Comment:
```suggestion
        final List<Money> moneys = ranks.stream()
                .map(Rank::getMoney)
                .collect(toList());
        EarningRate earningRate = Money.calculateEarningRate(moneys);
```

한 줄에 다 쓰면 코드 읽기가 힘듭니다.